### PR TITLE
Feature/iii 3445 optimize organizer search

### PR DESCRIPTION
**Pull request by simon-debruijn**
### Changed
- change boolean to use name instead of q param
---
Ticket: https://jira.uitdatabank.be/browse/III-3445

### CODE REVIEW

File: src/organizers/organizer-manager.service.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ function OrganizerManager(udbApi) {
    * @return {Promise.<PagedCollection>}
    */
   service.find = function(query, limit, start) {
-    return udbApi.findOrganisations(start, limit, null, query, true);
+    return udbApi.findOrganisations(start, limit, null, query, false);
```
Comment (simon-debruijn) [inline src/organizers/organizer-manager.service.js:35]: This boolean (useAdvancedQuery) effects the code below:

`if (useAdvancedQuery && query) {
      params.q = query;
  } else if (query) {
      params.name = query;
  }`

found in `udb-api.service.js`

Comment (bertramakers) [reply src/organizers/organizer-manager.service.js:35]: Thanks for the explanation 👍 
Do we still use this function with `useAdvancedQuery` set to `true`? If not we would remove that parameter and the if/else and just always use `params.name`

**Review by bertramakers — APPROVED** [review]
Added a small suggestion but technically this is good to merge! ✅

File: src/core/udb-api.service.spec.js [hunk 2]
```diff
@@ -467,7 +467,7 @@ describe('Service: UDB3 Api', function () {
     };
 
     $httpBackend
-      .expectGET(baseUrl + 'organizers/?embed=true&limit=10&q=foo-bar&start=0')
+      .expectGET(baseUrl + 'organizers/?embed=true&limit=10&name=foo-bar&start=0')
```
Comment (bertramakers) [inline src/core/udb-api.service.spec.js:470]: I think this test can just be removed. It's called `should find organizers via advanced query` (advanced query = q), and we already have a `should find organizers by a given name` which tests what you changed here.

Comment (bertramakers) [reply src/core/udb-api.service.spec.js:470]: Can you remove it in a follow-up PR?

Comment (simon-debruijn) [reply src/core/udb-api.service.spec.js:470]: Yep

Comment (simon-debruijn) [reply src/core/udb-api.service.spec.js:470]: https://github.com/cultuurnet/udb3-angular/pull/608

**Pull request merged**